JULIERAJ/KIDS-FIRST-v2
# Fix/124/toolbar btn error

## Remove console errors in the calendar

## Requirements
- Fix buttons error coming from _KFToolbar_
![Image](https://github.com/JULIERAJ/KIDS-FIRST-v2/assets/114008959/2c34225e-590c-4246-8002-e06fea85e900)
- Remove warning connected with onClose prop
![Image](https://github.com/JULIERAJ/KIDS-FIRST-v2/assets/114008959/ea104195-c121-46de-ade2-49bc8de789c7)

## Test Steps
- Click on the buttons "daily/weekly/monthly" and check that there is no warnings in the console
- Check there is no onClose prop warning in the console

## Notes
Merged Samah's branch and dev-back(just in case) into this branch to avoid conflicts.

## Checklist
- Pull Request title includes the issue number and a brief description of the change.
- All changes should be tested and verified to work correctly.
- Code follows frontend best practices.
- Branch names follow the conventions in the contributing file.
- Commit messages follow the naming conventions in the contributing file.
